com.authormore.Penpot-Desktop.yml

```diff
@@ -0,0 +1,83 @@
+app-id: com.authormore.Penpot-Desktop
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node20
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  # required to fix cursor scaling on wayland
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+
+  # D-Bus Access
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.ScreenSaver
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --talk-name=org.kde.kwalletd5
+  - --talk-name=org.kde.kwalletd6
+  # System D-Bus Access
+  - --system-talk-name=org.freedesktop.UPower
+  - --system-talk-name=org.freedesktop.login1
```

Comment:
are these actually required?

Reply:
Electron/zypak complaints otherwise, found the workarounds from Signal and vscode, not sure.

Reply:
can you figure out if these are needed? https://docs.flatpak.org/en/latest/debugging.html#audit-session-or-system-bus-traffic then go through the application

if it saves logins to keyrings `org.freedesktop.secrets, org.kde.kwalletd5 and org.kde.kwalletd6` are needed etc.

Reply:
Hmm, interesting
The logins seem to be local cookies, secrets are not needed
Last time it didn't even launch without the system bus, got many gpu errors, ozone initialization & bus errors but that was before dropping the x11 socket, but it works fine on wayland with all of them commented now :thinking:

Reply:
I still get 
> [3:1206/055912.474809:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory

which was how I found that issue but right now the app seems to function well regardless of that error,
'(required 1)' doesn't return anything, so let's drop I guess

Oh and technically, this app doesn't play audio anywhere yet, would dropping pulseaudio be too unexpected?

Reply:
> Oh and technically, this app doesn't play audio anywhere yet, would dropping pulseaudio be too unexpected?

No. drop it if not needed.

> [3:1206/055912.474809:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory

This is fine, it's not an error, electron does not cope with the dbus filter.

Reply:
more like electron tries to connect to it if it doesn't. When there is no system-bus permission the proxy doesn't create the socket and then the connection fail. (--log-system-bus doesn't either in that case)

TBF I'm not sure why Electron always try it, but when you turn a web browser into sonething you are likely to have a lot of bloat leftover.